python/tests/test_pyvw.py

```diff
@@ -122,13 +122,25 @@ def test_multilabel_prediction_type():
 
 def test_cbandits_label():
     model = vw(cb=4, quiet=True)
-    assert pyvw.cbandits_label(model.example('1 |')).costs[0].label == 1
+    cbl = pyvw.cbandits_label(model.example('1 |'))
```

Comment:
Since you are asserting all of the action cost and prob values below you should update this label with known values